atomics/t1127/t1127.yaml

```diff
@@ -0,0 +1,17 @@
+attack_technique: T1127
+display_name: Trusted Developer Utilities
+atomic_tests:
+- name: MSBuild Bypass Using Inline Tasks
+  description: |
+    Executes the code in a project file using. C# Example
+  supported_platforms:
+    - windows
+  input_arguments:
+    filename:
+      description: Location of the project file
+      type: Path
+      default: T1127.csproj
+  executor:
+    name: command_prompt
+    command: |
+     C:\Windows\Microsoft.NET\Framework\v4.0.30319\msbuild.exe T1127.csproj
```

Comment:
is this hardcoded to v4.0.30319 or can we figure out some way to determine whatever version of .net they have?

Reply:
Only this version is vulnerable to the Inline Task abuse.